tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 from .fixtures.metrics_fixtures import *
 from .fixtures.model_fixtures import *
 from .fixtures.movielens_fixtures import *
-from .fixtures.utils_fixtures import *
+# from .fixtures.utils_fixtures import *
```

Comment:
We'll probably want to un-comment this out.

Reply:
Excuse me I did not request re-review yet ;p